credoai/lens/lens.py

```diff
@@ -376,13 +376,20 @@ def _validate(self):
                     raise ValidationError(
                         "Sensitive features should have the same shape across assessment and training data"
                     )
+        if len(self.pipeline) == 0 and self.gov is None:
```

Comment:
`_validate` is running after Lens init, in theory we can have an empty pipeline and no governance, this is when users add steps manually.

Reply:
Yep, removed.